secboot/secboot_tpm.go

```diff
@@ -54,10 +54,34 @@ var (
 	sbAddEFISecureBootPolicyProfile  = sb.AddEFISecureBootPolicyProfile
 	sbAddSystemdEFIStubProfile       = sb.AddSystemdEFIStubProfile
 	sbAddSnapModelProfile            = sb.AddSnapModelProfile
+	sbProvisionStatus                = sb.ProvisionStatus
 	sbProvisionTPM                   = sb.ProvisionTPM
 	sbSealKeyToTPM                   = sb.SealKeyToTPM
 )
 
+// CheckTPMProvisionable verifies if the TPM can be provisioned, returning an error
+// if the TPM device does not exist, or if it exists and is already provisioned.
+func CheckTPMProvisionable() error {
+	errPrefix := "cannot check if the TPM can be provisioned"
+
+	tpm, err := sbConnectToDefaultTPM()
+	if err != nil {
+		return fmt.Errorf("%s: %v", errPrefix, err)
```

Comment:
Should we use something from golang.org/x/xerrors to wrap `err`? I think we are growing a convention around using it.

Reply:
we aren't growing a convention yet